timelock-server/src/main/java/com/palantir/atlasdb/timelock/lock/BlockingTimeouts.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.timelock.lock;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+import com.palantir.atlasdb.timelock.config.TimeLimiterConfiguration;
+import com.palantir.atlasdb.timelock.config.TimeLockServerConfiguration;
+
+import io.dropwizard.util.Duration;
+
+public final class BlockingTimeouts {
+    private static final Logger log = LoggerFactory.getLogger(BlockingTimeouts.class);
+
+    public static final long DEFAULT_IDLE_TIMEOUT = 30 * 1000;
+
+    private static final String SERVER = "server";
+    private static final String APPLICATION_CONNECTORS = "applicationConnectors";
```

Comment:
Anyway to deserialize using https://github.com/dropwizard/dropwizard/blob/4705f38cccddf9f15cb95c6f1db8c15a600c8078/dropwizard-core/src/main/java/io/dropwizard/server/DefaultServerFactory.java.